### PR TITLE
cli_pkg does not support version prefixed with v

### PR DESCRIPTION
**Pull request by Mogztter**

### CODE REVIEW

**Comment by Mogztter** [comment]
Seems to be working fine now: https://github.com/Mogztter/linkcheck/releases/tag/3.4.56
Everything is aligned, tag version, release version, changelog 👌🏻

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -2,8 +2,7 @@ name: Release
 
 on:
   push:
-    tags:
-      - 'v*' # Push events to matching v*, i.e. v1.0, v2.1.3
+    tags: '*'
```
Comment (filiph) [inline .github/workflows/release.yml:5]: I would prefer some way of distinguishing that this is really a version tag.

```suggestion
    tags: '*.*.*'
```

Something like that?

**Review by filiph — APPROVED** [review]
LGTM with one small suggestion.

File: .github/workflows/release.yml (hunk 1 above)
Comment (Mogztter) [reply .github/workflows/release.yml:5]: Sounds good!
Do you want to be even be more specific?

```suggestion
    tags: '[0-9]+.[0-9]+.[0-9]+'
```

Reference: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

**Pull request merged**